cnxepub/html_parsers.py

```diff
@@ -16,11 +18,13 @@
     }
 
 
-def _squash_to_text(elm):
+def _squash_to_text(elm, remove_namespaces=False):
     value = [elm.text or '']
     for child in elm.getchildren():
-        value.append(etree.tostring(child).decode('utf-8'))
+        value.append(etree.tostring(child).decode('utf-8').strip())
         value.append(child.tail or '')
+    if remove_namespaces:
+        value = [re.sub(' xmlns:?[^=]*="[^"]*"', '', v) for v in value]
```

Comment:
The problem with removing the namespace declarations like this is that we don't also remove the use of the namespace, we may end up with orphan tags. Admittedly, for our use-case the namespaces are almost always in the way - see my PR re: extending a bunch of xpaths to match both namespaced and non-namespaced (xhtml + html). The only trick I know for stripping them in a structured way is to create a root node with the default namespace set to what you want to remove, parse w/ ns_clean=True, then ... hehe I think it ends up using string manipulation in the end, to remove the default xmlns declaration. Hmm.